app/src/main/res/layout/connection_details_website_info.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/website_info_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <ImageView
+        android:id="@+id/details_back"
+        android:layout_width="@dimen/quicksettings_item_height"
+        android:layout_height="@dimen/quicksettings_item_height"
+        android:scaleType="center"
+        android:contentDescription="@string/etp_back_button_content_description"
+        app:tint="?attr/primaryText"
+        app:srcCompat="@drawable/mozac_ic_back" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:minHeight="@dimen/tracking_protection_item_height"
+            android:paddingTop="8dp"
+            android:paddingBottom="8dp">
+
+            <ImageView
+                android:id="@+id/favicon_image"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:importantForAccessibility="no"
+                android:scaleType="fitCenter"
+                android:layout_marginStart="16dp"
+                android:layout_gravity="center_vertical"
+                tools:drawable="@drawable/ic_internet" />
+
+            <TextView
+                android:id="@+id/title"
+                style="@style/QuickSettingsText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingTop="8dp"
+                android:paddingStart="8dp"
+                android:textStyle="bold"
+                tools:text="Wikipedia"
+                tools:ignore="RtlSymmetry" />
```

Comment:
We should try to avoid this since this means the title will run all the way to the right edge of the screen if the title is very long and just happens to fit in a single line. We should adding some padding to the end (we can change `paddingStart` to `paddingHorizontal`), but let's test with a long title.

Reply:
I'm just ignoring the issue as we already added `paddingStart` on the `style="@style/QuickSettingsText"`
https://github.com/mozilla-mobile/fenix/blob/9bfe9b07878eef70af0706c8708fc3393e739e76/app/src/main/res/values/styles.xml#L447, but the lint isn't able to see it.

Reply:
![image](https://user-images.githubusercontent.com/773158/125345375-75d3a880-e326-11eb-85fd-94867a6312c7.png)